gqcp/include/Basis/SpinorBasis/RSpinOrbitalBasis.hpp

```diff
@@ -92,6 +94,8 @@ class RSpinOrbitalBasis:
     // The type that represents a current density distribution for this spin-orbital basis.
     using CurrentDensityDistribution = EvaluableLinearCombination<complex, FunctionProduct<SpatialOrbital, SpatialOrbitalDerivative>>;
 
+    // TODO: Make DensityOperatorMatrixElement and CurrentDensityMatrixElement separate classes and bind them to Python.
+
```

Comment:
```suggestion

```

Is this "TO DO" still intentional? If so ignore it, otherwise you can commit this suggestion to delete it.

Reply:
Good catch: no it isn't! 👍🏻